cdap-docs/application-templates/source/etl/templates/sinks/batch/snapshotavro.rst

```diff
@@ -0,0 +1,60 @@
+.. meta::
+:author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+===============================
+Sinks: Batch: SnapshotAvro
+===============================
+
+.. rubric:: Description
+
+Snapshot sink for a FileSet that writes data in Avro format.
```

Comment:
A batch sink for a FileSet that writes a snapshot of data in an Avro format. [is this what you mean? we don't have "snapshot" sinks; just "batch" or "realtime"]